tests/sign.test.ts

```diff
@@ -55,3 +56,56 @@ test('Sign', async (t) => {
 
   t.end()
 })
+
+test('Sign: Inputted Verifying Keys', async (t) => {
+  const run = promiseRunner(t)
+
+  /* Setup Network */
+  await run('network up', spinNetworkUp(NETWORK_TYPE))
+  await sleep(process.env.GITHUB_WORKSPACE ? 30_000 : 5_000)
+  t.teardown(async () => {
+    // this gets called after all tests are run
+    await charlieStashEntropy.close()
+    await charlieEntropy.close()
+    await spinNetworkDown(NETWORK_TYPE).catch((error) =>
+      console.error('Error while spinning network down', error.message)
+    )
+  })
+
+  /* Setup Entropy */
+  await run('wasm', wasmGlobalsReady())
+  const charlieStashKeyring = new Keyring({ seed: charlieStashSeed, debug: true })
+  const charlieKeyring = new Keyring({ seed: charlieSeed, debug: true })
+  const charlieStashEntropy = new Entropy({
+    keyring: charlieStashKeyring,
+    endpoint: 'ws://127.0.0.1:9944',
+  })
+  const charlieEntropy = new Entropy({
+    keyring: charlieKeyring,
+    endpoint: 'ws://127.0.0.1:9944',
+  })
+  await run('charlieStashEntropy ready', charlieStashEntropy.ready)
+  await run('charlieEntropy ready', charlieEntropy.ready)
```

Comment:
This would make this test faster

```suggestion
  await Promise.all([
    run('charlieStashEntropy ready', charlieStashEntropy.ready),
    run('charlieEntropy ready', charlieEntropy.ready)
  ])
```